### DISCUSSION TITLE
[Tables][Transition] Transition on hover for .table-hover

### DISCUSSION THREAD

**Discussion by Martussky** [twbs/bootstrap#31594] [General]
Hi,

do you have an idea how to use the CSS transition for tr when table has a class `.table-hover` in BS 5? When I add for example `transition: 0.5s` to the code below it doesn't work?

```
.table-hover {
  > tbody > tr:hover {
    --bs-table-accent-bg: var(--bs-table-hover-bg);
    color: var(--bs-table-hover-color);
  }
}

```
I think this is caused by adding background-image to the cell here:

```
  > :not(caption) > * > * {
    padding: $table-cell-padding-y $table-cell-padding-x;
    background-color: var(--bs-table-bg);
    background-image: linear-gradient(var(--bs-table-accent-bg), var(--bs-table-accent-bg));
    border-bottom-width: $table-border-width;
  }
```

Thank you for your help!

**Comment by ggrund-tsi** [comment]
Hi @Martussky,

I had the same issue today. Transition don't work on 'tr' in BS 5.

Solution: Try to add the transition to 'td' instead.

`td {
        transition: 0.5s;
    }`

It worked for me.
Regards
